charts/rsyncd/.helmignore

```diff
@@ -14,9 +14,11 @@
 *.swp
 *.bak
 *.tmp
+*.orig
```

Comment:
What is it for? What kind of file is `.orig`? Remnant from your experiments?

If so,
```suggestion
```

Reply:
the file was copied from another chart, hence the change here. These files are created by multiple tools doing backups such as `sed` on OpenBSD or `git` on certain merge operation.

You don(t want these files in the packaged helm charts hence the presence here. Why was it absent before, I'm not sure.

I believe a subsequent PR with the same `.helmignore` file copied to all of our charts is in order WDYT?